operator/Makefile

```diff
@@ -201,7 +201,14 @@ GET_GITHUB_RELEASE_FN = get_github_release() { \
 	[ -x $${1} ] || { \
 		set -euxo pipefail ;\
 		mkdir -p bin ;\
-		curl --fail --location --output $${1} $${2} ;\
+		for i in {1..5}; do \
+			set +e ;\
+			curl --fail --location --output $${1} $${2} ;\
+			code=$$? ;\
+			set -e ;\
+			[ $$code -eq 0 ] && break ;\
+			[ $$i -eq 5 ] && exit 1; \
```

Comment:
This works, but might be a tiny maintainability problem: if someone wants to change the number of retries in the future and forgets to keep this `5` and the one in `{1..5}` in sync, ugly things could happen.

How about introducing an `attempts` variable and using it here, as well in the `for` line (over there you'd have to replace the `{..}` with a `seq` invocation, since the former seems to require literals.

Reply:
Done! Replaced `{..}` with `seq` and added `attempts` variable.